CMakeLists.txt

```diff
@@ -791,19 +861,18 @@ install(DIRECTORY ${CMAKE_BINARY_DIR}/perl/build/lib/ DESTINATION share/perl5
 install(DIRECTORY ${CMAKE_BINARY_DIR}/templates/blt/ DESTINATION share/git-core/templates)
 
 if(MSGFMT_EXE)
-	install(DIRECTORY ${CMAKE_BINARY_DIR}/locale DESTINATION share)
+	install(DIRECTORY ${CMAKE_BINARY_DIR}/po/build/locale DESTINATION share)
```

Comment:
This should probably be squashed into the first commit.

Reply:
okay